Jenkinsfile

```
@@ -54,6 +54,14 @@ def job = {
 
     if(params.CONFLUENT_PACKAGE_BASEURL) {
         override_config['confluent_common_repository_baseurl'] = params.CONFLUENT_PACKAGE_BASEURL
+    } else if (targetBranch().toString().endsWith('.x')) {
+        /* This condition imples we're in a dev (.x) branch and therefore the release in confluent_package_version 
+           does not yet exist on https://packages.confluent.io so we have to query the packaging job for the last
+           successful build location (what utilities.getLastNightlyPackagingBaseURL returns). We also override the
+           confluent_package_*_suffix to an empty string so it will install the (expected) latest version */
+        override_config['confluent_common_repository_baseurl'] = utilities.getLastNightlyPackagingBaseURL(targetBranch().toString())
```

```suggestion
        //override_config['confluent_common_repository_baseurl'] = utilities.getLastNightlyPackagingBaseURL(targetBranch().toString())
```

I'm re-evaluating things here - I only really have to do this because of the `*_suffix` variables locking things to install a specific version, If I just set these to be blank, then it should work w/o needing the location of the nightly packages.